src/IncludedCommands/UsefulFun/fly.lua

```diff
@@ -10,14 +10,15 @@ local Players = game:GetService("Players")
 local PhysicsService = game:GetService("PhysicsService")
 local RunService = game:GetService("RunService")
 local UserInputService = game:GetService("UserInputService")
+local ContextActionService = game:GetService("ContextActionService")
 
 local IncludedCommandUtil = require(script.Parent.Parent:WaitForChild("IncludedCommandUtil"))
 local Types = require(script.Parent.Parent.Parent:WaitForChild("Types"))
 
 return {
     Keyword = "fly",
     Category = "UsefulFunCommands",
-    Description = "Gives a set of players the ability to fly. Use E to toggle on/off.",
+    Description = "Gives a set of players the ability to fly. Use E on keyboard or "..UserInputService:GetStringForKeyCode(Enum.KeyCode.ButtonX):sub(7).." on controller to toggle on/off.",
```

Comment:
The rest of the project uses `string.sub`. I'd like it used for consistency.
```suggestion
    Description = "Gives a set of players the ability to fly. Use E on keyboard or "..string.sub(UserInputService:GetStringForKeyCode(Enum.KeyCode.ButtonX), 7).." on controller to toggle on/off.",
```